kfserving/kfserving-install/overlays/application/application.yaml

```diff
@@ -6,11 +6,11 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: kfserving-install
-      app.kubernetes.io/instance: kfserving-install-v0.7.0
+      app.kubernetes.io/instance: kfserving-install-v0.2.2
```

Comment:
Don't use mutable labels in selectors this will cause upgrade problems. You probably want to remove these from commonLabels. That could be one reason for the test failure since we have validation for that.

Reply:
Done.